store/tikv/coprocessor.go

```diff
@@ -421,13 +430,18 @@ func (it *copIterator) Next() ([]byte, error) {
 			return nil, nil
 		}
 	} else {
+		var closed bool
 		for {
 			if it.curr >= len(it.tasks) {
 				// Resp will be nil if iterator is finished.
 				return nil, nil
 			}
 			task := it.tasks[it.curr]
-			resp, ok = <-task.respChan
+			resp, ok, closed = recvFromRespCh(task.respChan, it.finished)
```

Comment:
s/closed/exit/ or s/exit/closed/

Reply:
I think **closed** is better, you see the comment line in the next line:
// **Close() is already called**, so Next() is invalid.

@zz-jason